### DISCUSSION TITLE
Only client doing requests is my router

### DISCUSSION THREAD

**Discussion by CtrlValCanc** [AdguardTeam/AdGuardHome#7201] [Q&A] [answered]
As title says, my router is the only device that seems to doing requests. It makes sense as I put Adguard Home's IP into my router's DNS settings. But when I was using another device for AdGuard Home I was able to see local ips of every device that connected to my network. I don't get what changed. I only have IPv4 from my ISP.
![cllient](https://github.com/user-attachments/assets/6d58260b-1e66-43d8-a7de-b81911d591a9)
I tried searching but I can't find a working solution for me. 
I have a FritBox 7530. I did not tweak much in AdGuard Home settings, excepet for adding blocklists.
Thanks for anyone that can help me.

**Comment by Dynamic5912** [comment]
I think for the FritzBox I read somewhere that it cannot do client IP's and all requests are forwarded from the router IP

**Reply by CtrlValCanc** [reply]
The weird thing is that on previous installation (I changed device from which I run AdGuard Home) and I had separate ip, that's why i find it weird

**Reply by Dynamic5912** [reply]
> The weird thing is that on previous installation (I changed device from which I run AdGuard Home) and I had separate ip, that's why i find it weird

Where have you set the AGH IP?

Under the WAN DNS or the LAN DNS on the FritzBox?

**Reply by CtrlValCanc** [reply]
In my router I have to go under Internet>Access data>DNS server, I don't think I have a difference between wan and lan

EDIT post marked as answer: I translated from my language to English, so the names of the settings were actually the ones you see in the first photo @ppfeufer posted.

**Reply by ppfeufer** [reply, marked as answer]
In a FritzBox, there might be 2 places where you have to set AGH as DNS, depending on if you use the FritzBox as DHCP or not.

### Internet » Type of Connection » DNS Server
![image](https://github.com/user-attachments/assets/c1f7e27e-0186-4b50-b84a-4d9bcd2d424b)
Set this to your AGH IP and untick the public DNS fallback.

### Home Network » Network » Network Settings » IPv4 Settings
![image](https://github.com/user-attachments/assets/92e8a655-94f0-4e04-8c25-1379c594d738)
Set the Local DNS Server to your AGH IP

### (Bonus for IPv6) Home Network » Network » Network Settings » IPv6 Settings
![image](https://github.com/user-attachments/assets/1975669c-150c-4810-bd35-ce830620159e)
Set your AGH IPv6 address as the local DNSv6 server

**Reply by CtrlValCanc** [reply]
oh! I did not had the DHCP corrected (in my defense, the setting was kind of hidden). 
I put my AGH local ip there, rebooted the router and now it works!!! That was my missing point!!
THANK YOU!!!
Do you also perchance now if there is a way to see the "local names" I see in the router interface instead of local ip only? 
If not, no problem, I'm so happy because I wasted hours for this problem and you fixed it!!!

**Reply by ppfeufer** [reply]
Ya, there are - AFAIK - 2 ways.

**A) Set your FritzBox as RDNS server in AGH's DNS settings (Settings » DNS Settings).**
![image](https://github.com/user-attachments/assets/0af4180e-5276-4a6e-8e8a-ad6e75167a56)
I cannot confirm that this works reliably, as I don't use it, the image is just an example.

**B) Use AGH's client configurations (Settings » Client Settings)**
This has the added benefit that you can name your clients as you wish there.
Bonus: In your FritzBox under Home Network » Network » Network Connections edit your clients and tell the FritzBox to always assign the same IP to this client. That way you pretty much reserve the client IP address and the FritzBox DHCP will always assign the same IP to the client based on their MAC address.
The result will be something like this
![image](https://github.com/user-attachments/assets/481536bb-21b9-4c8f-8ccc-7f57300547a9)
